index.js

```diff
@@ -5,11 +5,12 @@ const fs = require('fs');
 
 function run() {
   const lcovPath = core.getInput('path');
-  const minCoverage = core.getInput('min_coverage');
+  const minCoverageInput = core.getInput('min_coverage');
   const excluded = core.getInput('exclude');
   const excludedFiles = excluded.split(' ');
+  const minCoverage = parseMinCoverage(minCoverageInput);
 
-  if (!canParse(lcovPath)) {
+  if (minCoverage === null || !canParse(lcovPath)) {
```

Comment:
When can `minCoverage` be null? Shouldn't we also default to 100?

Reply:
On `parseMinCoverage(input)`, when the input is invalid, we call `core.setFailed` and return `null`. So, `minCoverage` can be null when the input is `'10%'`, `'-1'` and `'101'`.